source/pickAll.js

```diff
@@ -21,6 +21,7 @@ import _curry2 from './internal/_curry2';
  */
 var pickAll = _curry2(function pickAll(names, obj) {
   var result = {};
+  var obj = obj || {};
```

Comment:
While Ramda does not object to internal mutations or reassignments, I really don't like the idea of reassigning a parameter.  Could you alter the parameter name?